core/src/main/java/tc/oc/pgm/filters/dynamic/FilterMatchModule.java

```diff
@@ -497,5 +520,9 @@ public void onFlagStateChange(FlagStateChangeEvent event) {
     this.invalidate(match);
   }
 
+  public <T extends ReactorFactory.Reactor> T getReactorInstance(Class<T> reactorType) {
+    return (T) this.activeReactors.get(reactorType);
+  }
```

Comment:
```suggestion
  public <T extends ReactorFactory.Reactor> T getReactor(ReactorFactory factory) {
    return (T) this.activeReactors.get(factory);
  }
```

Feel free to make ReactorFactory a generic interface, so you can use the generic parameter there